test-it/ExpectedResults/TestVI_VariantOfClusterToArrayOfInnerType.vtr

```diff
@@ -0,0 +1,3 @@
+TestResult_ClusterOfDifferentSignednessNumericTypes_ConvertsCorrectly true
```

Comment:
I don't see tests that verify incompatible types report errors. That is an important test to have -- incompatible due to different array dimension and array data type. Also, one test that exercises the inflation codepath. I am guessing this is when the output terminal is unwired.

Reply:
Good point. Added a test for expected errors. Added one more test with cluster inside a cluster for non-error case. They are all passing.